api/v2/views/machine_request.py

```diff
@@ -40,10 +42,10 @@ def get_queryset(self):
                     Q(start_date__gt=timezone.now() - timedelta(days=7))
                 )
             )
-            if self.request.user.is_staff:
+            if request_user.is_staff:
```

Comment:
I chatted with Julian and PyCharm seems to be confirm that I am right that `request_user` is not a known system. Not sure if there is something missing here.